tests/integration/test_charm/lib/charms/velero_libs/v0/velero_backup_config.py

```diff
@@ -190,6 +197,12 @@ def __init__(
         )
         self.framework.observe(self._charm.on.upgrade_charm, self._send_data)
 
+        if refresh_event:
```

Comment:
I think would be great to have a functional test that actually check that the data-bag is updated in the refresh_events.